src/main/java/me/moodcat/soundcloud/SoundCloudExtract.java

```diff
@@ -13,6 +13,12 @@
  */
 public class SoundCloudExtract extends SoundCloudAPIConnector {
 
+    public static void main(String[] args) throws SoundCloudException {
```

Comment:
I suppose that this was a dev test? Maybe it should be removed or moved to a (integration) test case